### ISSUE TITLE
Trouble importing Paint Segmentation into Renderer

### ISSUE THREAD

**Issue by 10Cat10** [nci/drishti#18]
Hi, 
Each time I try to import the paint tags from Paint to Drishti Renderer the tags do not appear with seperate segmented colours or seperate transfer functions. 

Any ideas?
Thanks,
Cat.

**Comment by AjayLimaye** [comment]
Hi Cat,
 Load two volumes - tomogram and paint volume (no need to go through the
import phase for the paint volume).
.mask.pvl.nc file can be loaded directly into the renderer.

Use "mix tag" option to show the tags - volume 1 (i.e. mask volume) values
will be used to color the volume 0 region.  Modify color and opacity via
"Tag Colors" under "Preference" panel.

Cheers,
Ajay

**Comment by 10Cat10** [comment]
hi I have tried, but I don't know how to convert the mask file, it's not being imported into Drishti Renderer. and it doesn't have the file tag after it. It doesn't seem to be working.

**Comment by AjayLimaye** [comment]
Load .mask.pvl.nc file as the second volume.

**Comment by 10Cat10** [comment]
Thanks, but it seems to be still loading as problem with pvl.nc files and rendering them as lines? I presume the Drishti Paint makes a automatic mask file when you pull the pvl.nc file into paint?

**Comment by AjayLimaye** [comment]
Yes, .mask.pcl.nc and related .mask file is created when .pvl.nc file is
loaded in Paint.

**Comment by 10Cat10** [comment]
Yeah, it has done but it keeps coming up with errors, and everything works as lines, there's no actual mask data being rendered in Drishti.

**Comment by AjayLimaye** [comment]
Can you send me the mask files to check it here.
Cheers,
Ajay

**Comment by 10Cat10** [comment]
can I send you to your actual email? Just I don't want to share the files as they are new palaeontological data. Cheers. Thanks for the help.

**Comment by AjayLimaye** [comment]
Sure,  no problems.
Cheers,
Ajay

**Comment by 10Cat10** [comment]
could you email me on my work email? cl385?

**Comment by AjayLimaye** [comment]
Sorry, don't have your work email.
Sending data link via limaye.ajay@gmail.com is fine.

**Comment by 10Cat10** [comment]
It is all working fine, but I cannot find the mix tag function on the Tag Colours in the Preference section. Is it in the command box?

**Comment by AjayLimaye** [comment]
In the command box which is displayed after you press space bar.
